src/assets/icons/iconMap.js

```diff
@@ -0,0 +1,349 @@
+import { ReactComponent as Burger } from './burger.svg'
+import { ReactComponent as Calendar } from './calendar.svg'
+import { ReactComponent as Camera } from './camera.svg'
+import { ReactComponent as Cancel } from './cancel.svg'
+import { ReactComponent as CarMta } from './car-mta.svg'
+import { ReactComponent as CarUsage } from './car-usage.svg'
+import { ReactComponent as Car } from './car.svg'
+import { ReactComponent as Card } from './card.svg'
+import { ReactComponent as CaretLeft } from './caret-left.svg'
+import { ReactComponent as CaretRight } from './caret-right.svg'
+import { ReactComponent as CaretUp } from './caret-up.svg'
+import { ReactComponent as Caret } from './caret.svg'
+import { ReactComponent as Charge } from './charge.svg'
+import { ReactComponent as ChatBubble } from './chat-bubble.svg'
+import { ReactComponent as Chat } from './chat.svg'
+import { ReactComponent as Checkout } from './checkout.svg'
+import { ReactComponent as City } from './city.svg'
+import { ReactComponent as Claim } from './claim.svg'
+import { ReactComponent as ClipboardCheck } from './clipboard-check.svg'
+import { ReactComponent as Clock } from './clock.svg'
+import { ReactComponent as ContactEmail } from './contact-email.svg'
+import { ReactComponent as Copy } from './copy.svg'
+import { ReactComponent as Crop } from './crop.svg'
+import { ReactComponent as Cross } from './cross.svg'
+import { ReactComponent as DashboardHigh } from './dashboard-high.svg'
+import { ReactComponent as DirectDebit } from './direct-debit.svg'
+import { ReactComponent as DocDownload } from './doc-download.svg'
+import { ReactComponent as DocUpload } from './doc-upload.svg'
+import { ReactComponent as Download } from './download.svg'
+import { ReactComponent as Earth } from './earth.svg'
+import { ReactComponent as Edit } from './edit.svg'
+import { ReactComponent as EmailFilled } from './email-filled.svg'
+import { ReactComponent as Engine } from './engine.svg'
+import { ReactComponent as Europe } from './europe.svg'
+import { ReactComponent as Expand } from './expand.svg'
+import { ReactComponent as Facebook } from './facebook.svg'
+import { ReactComponent as Fence } from './fence.svg'
+import { ReactComponent as FileBadge2 } from './file-badge-2.svg'
+import { ReactComponent as FileCabinet } from './file-cabinet.svg'
+import { ReactComponent as File } from './file.svg'
+import { ReactComponent as Fire } from './fire.svg'
+import { ReactComponent as Flag } from './flag.svg'
+import { ReactComponent as Folder } from './folder.svg'
+import { ReactComponent as FraudAlert } from './fraud-alert.svg'
+import { ReactComponent as Fuel } from './fuel.svg'
+import { ReactComponent as Globe } from './globe.svg'
+import { ReactComponent as GooglePlayBadge } from './google-play-badge.svg'
+import { ReactComponent as Gpay } from './gpay.svg'
+import { ReactComponent as HelpAboutus } from './help-aboutus.svg'
+import { ReactComponent as HelpBuypolicy } from './help-buypolicy.svg'
```

Comment:
Very minor but should this be `HelpBuyPolicy` in line with the format of `HelpManagePolicy` below?

Reply:
This is all generated by the script based on the component names we have on Figma so the PascalCase convert based on where the hyphen is. It doesn't matter that much considering that the name we need to use is the one in Figma anyway